README.md

```diff
@@ -1,14 +1,14 @@
 # purescript-nonempty
 
 [![Latest release](http://img.shields.io/bower/v/purescript-nonempty.svg)](https://github.com/purescript/purescript-nonempty/releases)
```

Comment:
Most core libraries have a http://img.shields.io/github/release/purescript/purescript-xxx.svg badge, should we do the same here?

Reply:
I think it should be added.